docs/launch-with/galaxy-launch-with.rst

```diff
@@ -11,29 +11,35 @@ While browsing Dockstore
+Launching a workflow on an existing Galaxy server
+-------------------------------------------------
+
+The button will bring up a dropdown window where you may select which Galaxy instance to launch the workflow into. A few default options such as the `UseGalaxy.org <https://usegalaxy.org/>`__ instances are provided by Dockstore, but if you need to launch into another instance, you may provide a custom URL. Further along in this tutorial, you will find how to generate a custom instance and provide the URL to Dockstore.
 
 .. figure:: /assets/images/docs/galaxy/launch_with_options.png
    :alt: Launch options for Galaxy
 
+Launching a workflow on a custom Galaxy instance on the cloud
+-------------------------------------------------------------
+
 You can use Terra to spin up a custom cloud environment for Galaxy. Terra has integrated Galaxy making it possible for users to spin up a custom Galaxy server backed by a cloud environment and dedicated resources. This feature is available on all "faces" of Terra, such as AnVIL and NHLBI BioData Catalyst. Apart from providing a rapid and customizable Galaxy cloud instance, Terra is able to access a rich data corpus hosted in the cloud, including many controlled-access datasets facilitating the user to securely carry out analyses using this data.
 
-To spin up a custom cloud environment on Terra, navigate to the Notebooks tab on any Workspace on `Terra <https://app.terra.bio/>`__ and click on the Create a Cloud Environment for Galaxy on the left.
+To spin up a custom cloud environment on Terra, navigate to the Notebooks tab on any Workspace on `Terra <https://app.terra.bio/>`__ and click on the "Create a Cloud Environment for Galaxy" on the left.
 
 .. figure:: /assets/images/docs/galaxy/create_galaxy_environment.png
    :alt: Create a Cloud Environment for Galaxy
 
-It takes 8-10 minutes for the cloud environment to launch. Once it is ready, you can click on the Galaxy button on the top right and then the Launch Galaxy button. Doing so will open the Galaxy page with custom cloud environment. You can copy the URL and paste it into the Dockstore dropdown window to launch a workflow from Dockstore into this cloud instance. For your custom URL to work properly, it must start with ``https://`` and end with ``/galaxy``.
+It takes 8-10 minutes for the cloud environment to launch. Once it is ready, you can click on the Galaxy button on the top right and then the Launch Galaxy button. Doing so will open the Galaxy page with a custom cloud environment. You can copy the URL and paste it into the Dockstore dropdown window to launch a workflow from Dockstore into this cloud instance. For your custom URL to work properly, it must start with ``https://`` and end with ``/galaxy``.
```

Comment:
Is the second sentence specific to Galaxy in Terra (I don't know)? I suspect not, and you might be able to generalize this to any Galaxy deployment.

Reply:
I'm afraid I don't understand this one. Are you referring to this part: `Once it is ready, you can click on the Galaxy button on the top right and then the Launch Galaxy button.` I think this part is specific to Terra in the sense that, that is how the UI will display the options. I haven't really explored any other custom cloud instances to know if this can be generalized.
What do you suggest I do?

Reply:
Ah, sorry I wasn't clear, I meant the "You can copy the URL..." part. E.g., if UCSC set up its own Galaxy instance, e.g., galaxy.ucsc.edu, then you might be able to just copy the url there as well, even though it may not end with `/galaxy`. 

This is unproven, but my theory is if you just copy the url, it will work (please verify).

Reply:
I don't know if we can test it currently. I tried with a couple of public galaxy servers, they are either defunct or don't seem to have a route for TRS import. for eg. http://cistrome.org/ap/root

I believe I picked up this part,  `it must start with ``https://`` and end with ``/galaxy``,` from the existing document before editing it. To make it more generalized, we can just get rid of it. If users are properly copy and pasting the URL from Terra, it should work! We need not make it that specific.

Reply:
You could try with the existing public servers (usegalaxy.org, etc.).

Reply:
Ohh, they all work!

Reply:
I removed the last line, I don't see a need for it. This makes it more generalized.